nexus-repository-p2/src/test/resources/org/sonatype/nexus/repository/p2/internal/metadata/artifacts.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
the same. what this file for?

Reply:
@DarthHater @bhamail

Reply:
I believe this is for a Unit Test (don't quote me on that), where we test for if the artifacts.xml file is changed correctly.